src/components/books/Book.js

```diff
@@ -1,31 +1,65 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
+import React from 'react';
 import PropTypes from 'prop-types';
-import IndividualBook from './IndividualBook';
+import { useDispatch } from 'react-redux';
+import { removeBookAsync } from '../../redux/books/booksSlice';
+import classes from './IndividualBook.module.css';
 
-function Book({ booksInfo, onRemove }) {
+const Book = ({
+  title, author, category, id,
+}) => {
+  const dispatch = useDispatch();
+
+  const randomProgress = Math.floor(Math.random() * 100);
+  const chapterProgress = `CHAPTER ${Math.floor(Math.random() * 20)}`;
+  const progresspercentage = `${randomProgress} %`;
+
+  const handleRemoveBook = () => {
+    dispatch(removeBookAsync(id));
+  };
   return (
-    <>
-      {booksInfo.map((b) => (
-        <IndividualBook
-          key={Math.random()}
-          id={b.id}
-          title={b.title}
-          author={b.author}
-          gunra={b.gunra}
-          onRemove={onRemove}
-        />
-      ))}
-    </>
+    <div className={classes['book-container']}>
+      <div>
+        <p className={classes.gunra}>{category}</p>
+        <h1>{title}</h1>
+        <p className={classes.author}>{author}</p>
+        <div>
+          <button type="button" className={classes['btn-remove']}>Comments</button>
+          <button type="button" className={classes['btn-remove']} onClick={handleRemoveBook}>
+            Remove
+          </button>
+          <button type="button" className={classes['btn-remove']}>Edit</button>
+        </div>
+      </div>
+      <div className={classes.design}>
+        <div className={classes.progress}>
+          <div className={classes['circle-wrap']}>
+            <div className={classes.circle}>
+              <svg>
+                <circle cx="35" cy="35" r="35" />
+                <circle cx="35" cy="35" r="35" />
+              </svg>
+            </div>
+          </div>
+          <div>
+            <h3>{progresspercentage}</h3>
+            <p>Completed</p>
+          </div>
+        </div>
+        <div className={classes.chapter}>
+          <p className={classes.current}>CURRENT CHAPTER</p>
+          <h3 className={classes['chapter-name']}>{chapterProgress}</h3>
+          <button type="button" className={classes['btn-update']}>UPDATE PROGRESS</button>
+        </div>
+      </div>
+    </div>
   );
-}
```

Comment:
- [ ] As you should stick to the design as much as possible like color, same font, spacing. Please make sure that your design stick to the design as much as possible. Please see the design below for reference.
### Your design
![his](https://github.com/jubaer919/bookstore/assets/81727988/2f490738-adbd-491f-af74-1512774ebd3d)


### Original design
![original](https://github.com/jubaer919/bookstore/assets/81727988/17f77c9f-1db3-47d9-9b57-b1f6b64f9d50)

Reply:
I don't have exact color they use css-variable